src/coord/src/catalog.rs

```diff
@@ -741,7 +743,9 @@ impl Catalog {
     ///
     /// Returns the catalog and a list of updates to builtin tables that
     /// describe the initial state of the catalog.
-    pub async fn open(config: Config<'_>) -> Result<(Catalog, Vec<BuiltinTableUpdate>), Error> {
+    pub async fn open(
+        config: Config<'_>,
+    ) -> Result<(Catalog, Vec<BuiltinTableUpdate>, Vec<GlobalId>), Error> {
```

Comment:
Update doccomment describing the return type (I think: "tables to truncate").

Reply:
Actually, more detail would help, as I incorrectly misunderstood what it meant. It is (importantly) "tables to truncate before applying the updates". Idk if this matters, but I would have put it as the second return, to more clearly suggest that it should be the  first rather than second thing to apply to the catalog. Maybe I'm making that up... :)